src/components/Kanban.tsx

```diff
@@ -191,11 +202,12 @@ const Kanban = ({
   const cards = React.useMemo(() => {
     const cards: Record<string, Result[]> = {};
     data.forEach((d) => {
-      const column = toCellValue({
-        value: d[columnKey],
-        defaultValue: `No ${columnKey}`,
-        uid: d[`${columnKey}-uid`]?.toString(),
-      });
+      const column =
+        toCellValue({
+          value: d[columnKey],
+          defaultValue: `No ${columnKey}`,
+          uid: d[`${columnKey}-uid`]?.toString(),
+        }) || `No ${columnKey}`;
```

Comment:
I'd define a `DEFAULT_FORMAT` then interpolate the `columnKey` in there. Someone will eventually ask to configure it